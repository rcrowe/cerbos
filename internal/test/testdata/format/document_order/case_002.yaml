# - replace single quotes with double quotes
apiVersion: "api.cerbos.dev/v1"
description: 'Common dynamic roles used within the Apatr app'
derivedRoles:
  name: apatr_common_roles
  variables:
    local:
      flagged_resource: "request.resource.attr.flagged"
  definitions:
    - name: "owner"
      parentRoles: ['user']
      condition:
        match:
          expr: 'request.resource.attr.owner == request.principal.id'