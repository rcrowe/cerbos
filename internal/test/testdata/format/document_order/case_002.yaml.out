# - replace single quotes with double quotes
# - CEL formatted variables & expressions
apiVersion: "api.cerbos.dev/v1"
description: "Common dynamic roles used within the Apatr app"
derivedRoles:
  name: apatr_common_roles
  variables:
    local:
      flagged_resource: "request.resource.attr.flagged"
      env: "development"
  definitions:
    - name: "owner"
      parentRoles: ["user"]
      condition:
        match:
          expr: "request.resource.attr.owner == request.principal.id"
